daisy_workflows/image_import/inspection/inspect_disk.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+print("Status: <serial-output key:'partition_scheme' value:'mbr'>")
```

Comment:
Minor: I assume this is dummy output? Maybe comment it out so the example is still there.

Reply:
sgtm